docs/usage/tips-widget-creation.md

```diff
@@ -0,0 +1,114 @@
+# Pro-tips on widget creation
+
+Previously, the creation of widgets has been covered. To help users being more confident in creating widgets, here are some details to master the widget creation.
+
+<a id="howto-section"></a>
+## How to choose the appropriate widget visualization for your use case?
+
+We can classify the widgets in 3 different types.
+
+### Single dimension widgets
+
+Use these widgets when you would like to display information about one single type of object (entity or relation).
+
+- **Widget visualizations: number, list, list (distribution), timeline, donuts, radar, map, bookmark, tree map.**
+- Use case example: view the amount of malware in platform (number widget), view the top 10 threat actor group target a specific country (distribution list widget), etc.
+
+### Multi dimension widgets
+
+Use these widgets if you would like to compare or have some insights about similar types of object (up to 5).
+
+- **Widget visualizations: line, area, heatmap, vertical bars.**
+- Use case example: view the amount of malware, intrusion sets, threat actor groups added in the course of last month in the platform (line or area widget).
+
+!!! warning "Type of object in widget"
+
```

Comment:
if you delete the line between the title Type of object in widget" and the text after, it will be displayer properly

Without removing the line:
![image](https://github.com/OpenCTI-Platform/docs/assets/7083405/ee802955-98cd-4e2c-a020-808922cb0815)

After removing the line:
![image](https://github.com/OpenCTI-Platform/docs/assets/7083405/af7b18cb-e8b2-473a-8efd-8e7ac9d24214)

Reply:
Ok, I've juste updated it